src/transport/gatt.ts

```diff
@@ -47,14 +48,22 @@ export async function connect(): Promise<RpcTransport> {
       };
 
       dev.addEventListener('gattserverdisconnected', cb);
+
     },
   });
 
-  let writable = new WritableStream({
+  let writableWithoutResponse = new WritableStream({
     write(chunk) {
       return char.writeValueWithoutResponse(chunk);
     },
   });
 
-  return { label, readable, writable };
+  let writableWithResponse = new WritableStream({
+    write(chunk) {
+      return char.writeValueWithResponse(chunk);
+    },
+  });
+
+  return { label, readable,
```

Comment:
Can we defer creation of the two writables until this ternary resolves, to avoid the spurious object creation?

Reply:
just made an update, how does that look?